lib/push.js

```diff
@@ -11,7 +11,7 @@ var ERR_NO_PUSH_CALLBACK = 'No Push Callback'
 var ERR_DATA_BUT_NO_KEYS = 'Data payload present but missing key(s)'
 var ERR_TOO_MANY_DEVICES = 'Too many devices connected to account'
 
-var LOG_OP_PUSH_TO_DEVICES = 'push.pushToDevices'
+var LOG_OP_PUSH_TO_DEVICES = 'push.sendPush'
```

Comment:
This didn't have to change but I thought it made sense for all the log messages to reflect the method that is doing the work; `pushToDevices` doesn't exist any more.